src/MOIWrapper.jl

```diff
@@ -364,56 +363,56 @@ function LQOI.get_termination_status(instance::Optimizer)
             # MIP search incomplete but there is a solution
             other = xprsmoi_stopstatus(instance.inner)
             if other == MOI.OtherError
-                return MOI.OtherLimit
+                return MOI.OTHER_LIMIT
             else 
                 return other
             end
 
         elseif stat_mip == XPR.MIP_Infeasible
             if XPR.hasdualray(instance.inner)
-                return MOI.Success
+                return MOI.INFEASIBLE
             else
-                return MOI.InfeasibleNoResult
+                return MOI.INFEASIBLE
             end
         elseif stat_mip == XPR.MIP_Optimal
-            return MOI.Success
+            return MOI.OPTIMAL
         elseif stat_mip == XPR.MIP_Unbounded
             if XPR.hasprimalray(instance.inner)
-                return MOI.Success
+                return MOI.INFEASIBLE_OR_UNBOUNDED
             else
-                return MOI.UnboundedNoResult
+                return MOI.INFEASIBLE_OR_UNBOUNDED

```

Comment:
Did this prove unboundedness? Or dual infeasibility?

Same comment with the branch.